packages/components/src/Multiselect/Multiselect.tsx

```diff
@@ -111,9 +111,9 @@ export const Multiselect = ({
         {...props}
       >
         {unselected.map((item: MultiSelectItemProps) => (
-          <ComboBox.Item key={item.id} id={item.id}>
+          <ComboBox.Option key={item.id} id={item.id}>
```

Comment:
I wonder if you should also change MultiSelect.Item to Multiselect.Option?

And I can't use Sections now in MultiSelect or?

And what about using also Section as selectable Item? (asking because it's used in the customer-search - doesn't mean I like this)

Reply:
@sebald

Reply:
Since multi select is not a real component yet, I would not touch it.

Selectable Section: we talked about it. I think this would require some research, if we want to allow this pattern. Also it's an enhancement, we can add it independent of a breaking change